src/sagemaker/environment_variables.py

```diff
@@ -0,0 +1,53 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""Accessors to retrieve environment variables for hosting containers."""
+
+from __future__ import absolute_import
+
+import logging
+from typing import Dict
+
+from sagemaker.jumpstart import utils as jumpstart_utils
+from sagemaker.jumpstart import artifacts
+
+logger = logging.getLogger(__name__)
+
+
+def retrieve_default(
+    region=None,
+    model_id=None,
+    model_version=None,
+) -> Dict[str, str]:
+    """Retrieves the default container environment variables for the model matching the arguments.
+
+    Args:
+        region (str): Optional. Region for which to retrieve default environment variables.
+            (Default: None).
+        model_id (str): Optional. JumpStart model ID of the JumpStart model for which to
+            retrieve the default environment variables. (Default: None).
+        model_version (str): Optional. Version of the JumpStart model for which to retrieve the
+            default environment variables. (Default: None).
```

Comment:
@mufaddal-rohawala the goal is to provide a unique interface for customer: for example, in the future they could use the same utility to retrieve hyper-parameters for JumpStart, 1P algorithms and default DLC-containers for example.

It mirrors the logic of `image_uris.retrieve`: it can be used with JumpStart parameters, but we would like it to be generic so that other teams can add the ability to retrieve hyperparameters dict for their use cases. Right now though, we are the only team using it so it looks like JumpStart-specific.

Would that work?

Reply:
Ok makes sense, thanks for explaining, 
maybe we don't mention `Jumpstart` in arg docstring then but I'l leave this for you guys decide.